Tools/PlatformWPE.cmake

```diff
@@ -1,24 +1,37 @@
-if (DEVELOPER_MODE OR ENABLE_MINIBROWSER)
+if (ENABLE_API_TESTS OR ENABLE_LAYOUT_TESTS OR ENABLE_MINIBROWSER)
     add_subdirectory(wpe/backends)
 endif ()
 
-if (DEVELOPER_MODE)
+if (ENABLE_LAYOUT_TESTS)
     add_subdirectory(ImageDiff)
     add_subdirectory(TestRunnerShared)
     add_subdirectory(WebKitTestRunner)
+endif ()
+
+if (DEVELOPER_MODE)
     add_subdirectory(flatpak)
+endif ()
 
-    if (ENABLE_API_TESTS)
-        add_subdirectory(TestWebKitAPI/glib)
-    endif ()
+if (ENABLE_API_TESTS)
+    add_subdirectory(TestWebKitAPI/glib)
 endif ()
 
 if (ENABLE_MINIBROWSER)
     add_subdirectory(MiniBrowser/wpe)
 endif ()
 
-if (DEVELOPER_MODE AND ENABLE_COG)
+if (ENABLE_COG)
     include(ExternalProject)
+    if ("${WPE_COG_PLATFORMS}" STREQUAL "")
```

Comment:
I think this should be exposed as a string-valued option. What you have here is basically a secret only for those who know to use it.

Reply:
Yes, it seems it will be more clean doing it that way
I can give it a try

Reply:
Although I see now there are other preexisting "options" that should match: `WPE_COG_REPO` and `WPE_COG_TAG`.

Reply:
As far as I can see all the CMake variables that we expose are all boolean. I can't see another example of a a CMake string variable exposed in WebKit. And it doesn't look very right to start doing that just for this.

Do you know about any example of a string variable exposed? The closest I could find is the CMake variable `USER_AGENT_BRANDING` but that one is also hidden to the build

Reply:
So it seems CMake options are always bool value. Ignore me, then....